dmrunner/process.py

```diff
@@ -28,7 +27,6 @@ def _get_clean_env(self):
             del env['VIRTUAL_ENV']
 
         env['PYTHONUNBUFFERED'] = '1'
-        env['DMRUNNER_USER'] = getpass.getuser()
```

Comment:
There are a couple of files in `config` (`config/docker-compose.yml` and `config/populate_db.sh`) that depend on this variable, so these will need updating if we're going to drop this custom variable.

Reply:
Mmm, I'm getting `role "samuelwilliams" does not exist` even with this updated. 🤔

Reply:
Sorry, that shouldn't be in this commit, that was something else I was working on.

I'll remove it from this PR, I think the role error should disappear if you rebuild the docker container.

Reply:
I was rebuilding the container, still had it. Could've been something else on my end though. 🤷‍♂️